modules/rstudio/hm.nix

```
@@ -0,0 +1,26 @@
+{
+  config,
+  lib,
+  ...
+}:
+{
+  options.stylix.targets.rstudio = {
+    enable = config.lib.stylix.mkEnableTarget "RStudio" false;
+  };
```

Consider doing the following:

```suggestion
  options.stylix.targets.rstudio.enable =
    config.lib.stylix.mkEnableTarget "RStudio" false;
  };
```